### DISCUSSION TITLE
Make the app archive filter to `show all` by default

### DISCUSSION THREAD

**Discussion by ghost** [directus/directus#17158] [Feature Requests]
Make the app archive filter to `show all` by default, otherwise users maybe never know that there is an archive.

**Comment by rijkvanzanten** [comment]
I'm not quite sure if I understand what you mean by "maybe never know that there is an archive."

The point of the archiving currently is to act as a soft-delete, where archived items are hidden from the regular view. 🤔

**Reply by ghost** [reply]
What I mean is, that by default items disappear after being archived. And my concerns are that clients/editors/users don’t get it.

My suggestion is to make it `Show All Items` by default.

<img width="280" alt="img" src="https://user-images.githubusercontent.com/2291224/213490200-2c0a64a9-68fb-4f5e-bb85-c2624de23f07.png">

OR even better: make it an option in the project settings:

<img width="825" alt="is" src="https://user-images.githubusercontent.com/2291224/213490790-7cd3bd7a-65af-467f-a890-619be3243112.png">

<img width="413" alt="could" src="https://user-images.githubusercontent.com/2291224/213491464-cf67d9d8-b721-49c8-a357-645b80db3787.png">

**like this ↑**

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
